### DISCUSSION TITLE
Ethereum Mainnet disk space requirements

### DISCUSSION THREAD

**Discussion by mitro** [blockscout/blockscout#10748] [Support]
We are synchronizing Blockscount with Ethereum Mainnet and need to plan how much disk space is required. The docs state that it is 11 TB https://docs.blockscout.com/setup-and-run-blockscout/requirements/database-storage-requirements

Our observations differ from the docs. We have 1,5 mln blocks synchronized, and it took 2.8 TB already. If we extrapolate to 20.5 mln blocks, the required disk space will be about 40 TB. The early blocks contain fewer transactions than the later ones, so we can reduce the estimation. However, I think it will still be more than 30 TB.

Can you provide the exact amount of disk space used for the fully synchronized Ethernet Mainnet?

**Comment by vbaranov** [comment]
The current storage size for the Ethereum Mainnet instance is 21 TB. [The corresponding docs page](https://docs.blockscout.com/setup/requirements/database-storage-requirements) has been updated. Thanks for flagging this.
